systems/controllers/QPCommon.cpp

```diff
@@ -367,7 +367,7 @@ int setupAndSolveQP(NewQPControllerData *pdata, std::shared_ptr<drake::lcmt_qp_c
 
   MatrixXd R_DQyD_ls = R_ls + D_ls.transpose()*Qy*D_ls;
 
-  pdata->r->doKinematics(robot_state.q,false,robot_state.qd);
+  pdata->r->doKinematics(robot_state.q,true,robot_state.qd);
```

Comment:
@kuindersma , in the QP controller, it computes Jcomdot by calling getCOMJacDot and stores it in the pdata. Can we remove the Jdot stored in pdata? I do not think we really need Jdot in this case. What we really need is JdotTimesV, and it is implemented in new doKinematics without calling the second order gradient.